manifests/prepare/package.pp

```diff
@@ -24,10 +24,15 @@
       }
       $source = "puppet:///pe_packages/${pe_server_version}/${tag}/${package_file_name}"
     } elsif $::operatingsystem == 'AIX' {
-      $tag = $::platform_tag ? {
-        'aix-7.2-power' => 'aix-7.1-power',
-        default         => $::platform_tag,
+      if $package_file_name =~ /^puppet-agent-6.*/ {
```

Comment:
It seems like there's a better way to check this here -- is there a way I could check the collection as opposed to the agent version?

Reply:
come to think of it, I don't expect this will even work until the reported agent version is updated

Reply:
I _think_ you can use `$::puppet_agent::collection`